services/workflows-service/scripts/workflows/dynamic-ui-workflow.ts

```diff
@@ -104,6 +105,17 @@ export const dynamicUiWorkflowDefinition = {
             },
             { target: 'pending_kyc_response_to_finish' },
           ],
+          KYC_RESPONDED: [
```

Comment:
This one should be reverted i think, since per Nitzan, while there a document in revision we should stay in revision regardless of what happens with the KYC